khelo/lib/ui/flow/team/add_team_member/add_team_member_screen.dart

```diff
@@ -186,37 +221,75 @@ class _AddTeamMemberScreenState extends ConsumerState<AddTeamMemberScreen> {
     BuildContext context,
     AddTeamMemberState state,
   ) {
+    return SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      padding: const EdgeInsets.symmetric(horizontal: 16),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          for (final user in state.selectedUsers) ...[
```

Comment:
I think you need to use listview with horizontal scroll,map instead of for loop.

Let's avoid for loop in view if available other options.

Reply:
for listview-change, I think SingleChildScrollview will be better as with ListView i need to give some specific Height as it'll be inside Column.